composer.json

```diff
@@ -32,7 +32,7 @@
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "gedmo/doctrine-extensions": "~2.4",
         "symfony-cmf/routing-bundle": "1.2.*",
-        "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony/symfony": "^2.8.7, !=2.8.10 || ^3.0",
```

Comment:
symfony 3 is currently not possible because of symfony-mf/routing-bundle see PR https://github.com/sulu/sulu/pull/3154